source/client/benchmark_client_impl.cc

```diff
@@ -169,5 +207,22 @@ void BenchmarkClientHttpImpl::onPoolFailure(Envoy::Http::ConnectionPool::PoolFai
   }
 }
 
+void BenchmarkClientHttpImpl::exportLatency(const uint32_t response_code,
+                                            const uint64_t latency_ns) {
+  if (response_code > 99 && response_code <= 199) {
```

Comment:
optional:  it seems very odd to me to notate this expression this way.  It's more conventional to do:
response_code >= 100 && response_code < 200

Reply:
ah, I was following the code here.
https://github.com/envoyproxy/nighthawk/blob/c57811f3626b53f997f0241aaa482366dfe790c0/source/client/benchmark_client_impl.cc#L140-L152